classes/book.rb

```diff
@@ -0,0 +1,15 @@
+require './item'
+
+class Book < item
```

Comment:
There seems to be a little mistake where you `Book `extends `item` instead of `Item`. Kindly change it because `Item `is a class and by convention should be capitalized. 😎